spread.yaml

```diff
@@ -612,33 +627,20 @@ suites:
         # Test cases are not yet ported to Fedora/openSUSE that is why
         # we keep them disabled. A later PR will enable most tests and
         # drop this blacklist.
-        systems: [-fedora-*, -opensuse-*, -amazon-*, -arch-*]
+        backends: [google-nested]
         environment:
-            NESTED_TYPE: "$(HOST: echo ${SPREAD_NESTED_TYPE:-core})"
             NESTED_ARCH: "$(HOST: echo ${SPREAD_NESTED_ARCH:-amd64})"
             NESTED_SYSTEM: "$(HOST: echo ${SPREAD_NESTED_SYSTEM:-xenial})"
             CORE_REFRESH_CHANNEL: "$(HOST: echo ${SPREAD_CORE_REFRESH_CHANNEL:-candidate})"
-        halt-timeout: 2h
-        kill-timeout: 2h
         manual: true
         prepare: |
             #shellcheck source=tests/lib/pkgdb.sh
             . "$TESTSLIB"/pkgdb.sh
             distro_update_package_db
-            distro_install_package snapd qemu genisoimage sshpass
-            if [ "$NESTED_TYPE" = "core" ]; then
-                snap install --classic --beta ubuntu-image
-            else
-                distro_install_package qemu-kvm cloud-image-utils
-            fi
+            distro_install_package snapd qemu genisoimage sshpass qemu-kvm cloud-image-utils ubuntu-image
```

Comment:
Do we need to install snapd package given that we inject our local build soon after?

Reply:
Yes, because there are other tests which are not building snapd and require it.